Content.Server/Objectives/Systems/NotCommandRequirementSystem.cs

```diff
@@ -20,8 +19,10 @@ private void OnCheck(EntityUid uid, NotCommandRequirementComponent comp, ref Req
         if (args.Cancelled)
             return;
 
-        // cheap equivalent to checking that job department is command, since all command members require admin notification when leaving
-        if (_job.MindTryGetJob(args.MindId, out _, out var prototype) && prototype.RequireAdminNotify)
-            args.Cancelled = true;
+        if (TryComp<MindComponent>(args.MindId, out var mind) && mind.OwnedEntity != null)
+        {
+            if (HasComp<CommandStaffComponent>(mind.OwnedEntity.Value))
+                args.Cancelled = true;
+        }
```

Comment:
`if (args.Mind.OwnedEntity is {} ent && HasComp<CommandStaffComponent>(ent))`

Reply:
thanks, i'm still new with the c#